usecases/delegated.go

```diff
@@ -84,6 +86,8 @@ func (u DelegatedUserInterface) PostSnippet(author types.UserId, contents string
 		return types.Snippet{}, err
 	}
 
+	u.Highlighter.Post(snippet)
```

Comment:
What will happen when the buffer in the buffered channel be full?

Reply:
Added a warning, don't really know what to do besides that.

Reply:
It is actually a design fault. You need an actual queue between API call and Highlighter. The easiest way to do it is to store an event into DB at the API call and reading events from DB with some goroutine (actual pipeline start).